langlib/lang.runtime/src/main/ballerina/runtime.bal

```diff
@@ -41,6 +41,16 @@ type CallStackElement record {|
 # The listener becomes a module listener of the module from which this
 # function is called.
 #
+# ```ballerina
+# runtime:DynamicListener ln = object {
+#   public function 'start() returns error? {}
```

Comment:
Formatting looks incorrect.

Reply:
Meant the indentation. One-line form here is OK. Please revert the change and fix indentation.

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-lang/pull/39006/commits/0c0e021a6e4f631cf924cdff82704d3514345a25